cmd/build/test.go

```diff
@@ -165,21 +169,23 @@ func (t Test) Integration(ctx context.Context, self run.DependencyIDer, filter s
 
 // Run unittests, the filter argument is passed via -run="".
 func (t Test) Unit(_ context.Context, filter string) error {
-	if err := os.MkdirAll(".cache/unit", 0o755); err != nil {
+	if err := os.MkdirAll(filepath.Join(".cache", "unit"), 0o755); err != nil {
 		return err
 	}
 
-	gotestArgs := []string{"-coverprofile=.cache/unit/cover.txt", "-race", "-json"}
+	gotestArgs := []string{"-coverprofile=" + filepath.Join(".cache", "unit", "cover.txt"), "-race", "-json"}
 	if len(filter) > 0 {
 		gotestArgs = append(gotestArgs, "-run="+filter)
 	}
 
 	argStr := strings.Join(gotestArgs, " ")
+	tstPath := dotFilepathJoin("...")
```

Comment:
Not a file path, only use forward slashes

Reply:
Fixed